environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/response/EnvironmentStatus.java

```diff
@@ -37,6 +37,9 @@
     PUBLICKEY_CREATE_IN_PROGRESS("Creating SSH Public key"),
     PUBLICKEY_DELETE_IN_PROGRESS("Deleting SSH Public key"),
 
+    ENVIRONMENT_RESOURCE_ENCRYPTION_INITIALIZATION_IN_PROGRESS("Initializing Resource Encryption settings for environment"),
```

Comment:
this is probably requires ui update

Reply:
Filed https://jira.cloudera.com/browse/CB-11791 to track UI updates, cc: @subhashjha35